integration-tests/itest/deploy_test.go

```diff
@@ -0,0 +1,306 @@
+package main_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
+	"time"
+
+	"os/exec"
+
+	"lightbend.com/cloudflow/itest/cli"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// Assumptions:
+// In this test, we are going to deploy the test application to an existing cluster.
+// For this first step, we assume that the cluster is correctly deployed and that kubectl is configured to talk to it.
+// The application is already built
+// We also assume that the test is going to be executed manually.
+// Automation will be the next step
+
+const (
+	JsonTokenSrc    = "/keybase/team/assassins/gcloud/pipelines-serviceaccount-key-container-registry-read-write.json"
+	ShortTimeout    = 60  // seconds
+	LongTimeout     = 240 // seconds
+	InitialWaitTime = "30s"
+)
+
+var swissKnifeApp = cli.App{
+	Image: "eu.gcr.io/bubbly-observer-178213/swiss-knife:189-277e424",
```

Comment:
I think we should publish this in a public docker registry (e.g. the lightbend dockerhub?) so no credentials are required for pulling and non-team members can also run these tests.

Reply:
good idea!

Reply:
done.